data/api/client-server/pusher.yaml

```diff
@@ -139,6 +139,11 @@ paths:
         This endpoint allows the creation, modification and deletion of [pushers](/client-server-api/#push-notifications)
         for this user ID. The behaviour of this endpoint varies depending on the
         values in the JSON body.
+
+        If `kind` is not `null`, the pusher with this `app_id` and `pushkey`
```

Comment:
we're probably supposed to change the `type` of `kind`. In OpenAPIv3 I think that means adding `nullable: true`, but it might be harder in v2? Maybe just add a comment for now?

Reply:
There are already some uses of `null` in the current definitions, like this:

```yaml
type:
    - "string"
    - "null"
```

OpenAPI v3 does use `nullable: true`, but v3.1 uses the same syntax as above.